src/ZulipMobile.js

```diff
@@ -1,4 +1,5 @@
 /* @flow */
+import './ReactotronConfig';
```

Comment:
Reactotron is needed to be imported in first line of your project if not done so sometimes due to changes in ordering of imports it results in errors.
Refrences:-
Docs- https://github.com/infinitered/reactotron/blob/v1.x/docs/quick-start-react-native.md 
Checkout importing reactotron portion
Commmon Errors- https://github.com/infinitered/reactotron/issues/289 
P.S. - I did spend a day or two with the same so I guess its better for all of us to keep it that way.